doc/source/whatsnew/v1.1.0.rst

```diff
@@ -901,6 +901,7 @@ I/O
 - Bug in :meth:`~DataFrame.read_feather` was raising an `ArrowIOError` when reading an s3 or http file path (:issue:`29055`)
 - Bug in :meth:`~DataFrame.to_excel` could not handle the column name `render` and was raising an ``KeyError`` (:issue:`34331`)
 - Bug in :meth:`~SQLDatabase.execute` was raising a ``ProgrammingError`` for some DB-API drivers when the SQL statement contained the `%` character and no parameters were present (:issue:`34211`)
+- Bug in :meth:`DataFrame.to_sql` when reading DataFrames with `-np.inf` entries (:issue: `34431`)
```

Comment:
-np.inf needs a double backtick. Also there is a space between :issue: and the issue number

Reply:
Fixed these!